index.bs

```diff
@@ -1213,6 +1209,52 @@ Advisory Board and Technical Architecture Group Elections</h5>
 	Refer to <a href="https://www.w3.org/2002/10/election-howto">How to Organize an Advisory Board or TAG election</a> [[ELECTION-HOWTO]]
 	for more details.
 
+<h5 id="TAG-appointments">
+Technical Architecture Group Appointments</h5>
+
+	The [=Team=] is responsible for appointing
+	3 of the participants to the [=Technical Architecture Group=].
```

Comment:
Does the Team _have_ to apoint three participants?

Reply:
Good question.
I think they ought to try, but they could fail to find the right number of suitable people, or, possibly more importantly, they could fail to get them all ratified… So I don't think we can strictly mandate it. I think the current wording works. It combines:
* the fact that the Team is responsible for getting 3
* the fact that vacant seats *should* be filled
* the fact that the team *should* get their appointment ratified before the start of the term